crm/customizations.rst

```diff
@@ -15,6 +15,7 @@ Customizations
    customizations/crm_multicompany
    customizations/crm_task
    customizations/crm_offers
+   customizations/mail_activity_systray_for_crm
```

Comment:
You are using incorrect module name. Its called `mail_activity_systray_crm`

Reply:
Module is mail_activity_systray_crm, but it is called Mail Activity Systray for CRM.